### PR TITLE
Sara/mkt 896 audit faq

### PR DESCRIPTION
**Pull request by s-santillan**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [ ] A technical writer reviews the content or PR

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -197,6 +216,6 @@ A policy is a simple collection of rules and a definition of what to do with rul
 
 ### Do you have a visualization UI?
 
-Semgrep Team users can create custom dashboards and visualizations. Semgrep also supports posting results through [webhooks](https://semgrep.dev/docs/semgrep-cloud-platform/webhooks) to any JSON endpoint, so you can easily integrate it with your favorite visualization tool.
+Semgrep Cloud Platform users can create custom dashboards and visualizations. Semgrep also supports posting results through [webhooks](https://semgrep.dev/docs/semgrep-cloud-platform/webhooks) to any JSON endpoint, so you can easily integrate it with your favorite visualization tool.
```
Comment (pabloest) [inline docs/faq.md:219]: Not Semgrep AppSec Platform as you changed above?

File: docs/faq.md [hunk 2]
```diff
@@ -187,7 +206,7 @@ Semgrep CI makes network requests in accordance with the data storage previously
 
 ### How do I configure Semgrep for different projects?
 
-Semgrep App provides centralized policy management. See the [Policies documentation](/semgrep-code/policies/) for more details.
+Semgrep Cloud Platform provides centralized policy management. See the [Policies documentation](/semgrep-code/policies/) for more details.
```
Comment (pabloest) [inline docs/faq.md:209]: Not Semgrep AppSec Platform as you changed above?

File: docs/faq.md [hunk 3]
```diff
@@ -160,13 +176,16 @@ All customer data is located in AWS (US region). Amazon RDS encrypted database i
...
-No. Semgrep CI runs fully in your CI pipeline and your source-code never leaves your environment. Only meta-data related to Semgrep runs (see below) are sent to Semgrep's service.
+
+By default, Semgrep configurations run fully in your CI pipeline and your source code never leaves your environment. Only metadata related to Semgrep runs (see the following question) are sent to Semgrep's service.
+
+If you choose to enable it, Semgrep Assistant requires code access. See the [Privacy and legal considerations](/semgrep-code/semgrep-assistant-code/#privacy-and-legal-considerations) section to understand how your code is stored and retained.
 
 ### What data is stored?
 
-[Semgrep](https://github.com/semgrep/semgrep) may send data to Semgrep App in accordance with the [metrics policy](/metrics).
+[Semgrep](https://github.com/semgrep/semgrep) sends data to Semgrep Cloud Platform in accordance with the [metrics policy](/metrics).
```
Comment (pabloest) [inline docs/faq.md:186]: Not Semgrep AppSec Platform as you changed above?

**Review by pabloest — APPROVED** [review]

File: docs/faq.md (hunk 1 above)
Comment (s-santillan) [reply docs/faq.md:219]: Hi, I realized that we haven't updated any of our collateral yet around calling it an Appsec platform. Maybe I should stick to SCP first?

**Review by khorne3 — APPROVED** [review]
Maybe we leave SCP as is in this PR and address all name changes in a subsequent PR?

**Pull request merged**